en/micro-integrator/docs/administer-and-observe/using-the-analytics-dashboard.md

```diff
@@ -67,12 +67,12 @@ Let's start the servers in the given order.
 !!! Warning
     Be sure to [configure the SSL certificates](../../setup/observability/setting-up-classic-observability-deployment/#configure-ssl-certificates) before starting the Micro Integrator.
 
-Once you have [started the Analytics Worker](#starting-the-analytics-worker), you can [start the Micro Integrator](../../../setup/installation/install_in_vm_installer/).
+Once you have [started the Analytics Server](#starting-the-analytics-worker), you can [start the Micro Integrator](../../../setup/installation/install_in_vm_installer/).
```

Comment:
We need to change the "#starting-the-analytics-worker" part as well right?

Reply:
fixed